### PR TITLE
Include language for CHT

### PR DESCRIPTION
**Pull request by abbyad**
There are many changes needed throughout the repo if we were to switch everything to CHT language. The purpose of this PR is to bring in language for the transitional state so that it makes sense with were we are at now -- having a `medic` repo as the Community Health Application Framework.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 # Medic Mobile
 
-These instructions should help you get setup to run or develop on Medic Mobile.
-For latest changes and release announcements see our [change log](Changes.md).
+These instructions should help you get setup to run or develop on Medic Mobile's Community Health Application Framework. For latest changes and release announcements see our [change log](Changes.md).
```
Comment (garethbowen) [inline README.md]: Changes.md is the old location - now it just has a link to the new location. I think change this to...

`...[release notes](https://github.com/medic/medic/tree/master/release-notes).`

**Comment by garethbowen** [comment]
I have no idea why those tests are failing for you!

**Comment by abbyad** [comment]
Thanks, just made the change.

**Comment by abbyad** [comment]
Note that only 1 file was changed, not 260 as stated in the PR. A rebase was done post review to try to get the build to pass. As it turns out, a timeout in a test was passed eventually by restarting the job so the rebase was unnecessary. Ultimately the diff of only 1 file is seen in the final squash and merge commit: https://github.com/medic/medic/commit/d420e2fa4d9a2cec35f24b43e26253a5a0f8083f

**Pull request merged**